src/organisation/containers/update-pba-numbers/update-pba-numbers.component.html

```diff
@@ -0,0 +1,32 @@
+<app-hmcts-main-wrapper [backLink]="'/organisation'" [title]="title">
```

Comment:
title=title?

Reply:
I decided to put the title in the TS file, it is a variable reference